app/models/user.rb

```diff
@@ -16,4 +16,8 @@ def role_names_for(application: )
   def roles
     memberships.map(&:roles).flatten
   end
+
+  def is_webops?
```

Comment:
Exactly. Also webops is a special role that can do anything do it makes sense to be able to shortcut the other permission checks if the user is webops